src/main/java/view/BlockType.java

```diff
@@ -0,0 +1,33 @@
+package view;
+
+import domain.Line;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum BlockType {
+    CROSS("-", true),
+    NOT_CROSS(" ", false);
+
+    private final String type;
+    private final boolean isCross;
+
+    private BlockType(String type, boolean isCross) {
+        this.type = type;
+        this.isCross = isCross;
+    }
+
+    public static List<String> getBlockTypes(Line line) {
+        List<Boolean> blockLine = line.getLine();
+
+        return blockLine.stream()
+                .map(BlockType::decideBlockType)
+                .collect(Collectors.toList());
+    }
+
+    private static String decideBlockType(boolean block) {
+        if (block == CROSS.isCross) {
+            return CROSS.type;
+        }
+        return NOT_CROSS.type;
+    }
```

Comment:
힌트입니다 :) 이렇게 변경하고 변경 여파를 처리해보시고, 지금 방식과 어떻게 다른지 확인해주셔요
```suggestion
    public static List<BlockType> getBlockTypes(Line line) {
        List<Boolean> blockLine = line.getLine();

        return blockLine.stream()
                .map(BlockType::decideBlockType)
                .collect(Collectors.toList());
    }

    private static BlockType decideBlockType(boolean block) {
        if (block == CROSS.isCross) {
            return CROSS;
        }
        return NOT_CROSS;
    }

    public String getType() {
        return type;
    }
```

Reply:
현재에는 값으로 된 리스트로 전달해주는 방식이었는데 이렇게 하면 나중에 enum이 확장되는 경우 ex) 필드가 추가되는 경우에 새로운 메소드를 만들어야하는 점이 확장에서 대해서 대응력이 떨어지는 것 같습니다. 제시하신 방법은 객체를 전달해주어 view에서 값을 getter를 통해서 가져오는 방법이었고 이는 변화에 대응하기 편하고 코드의 재사용 가능성이 높다는 것을 깨달을 수 있었습니다.